core_lib/web_helpers/request_response_helpers.py

```diff
@@ -45,9 +45,9 @@ def generate_response(data, status, media_type: MediaType, headers: dict = {}):
         data = b''
     elif media_type == MediaType.APPLICATION_JSON:
         data = json.dumps(data)
-    if WebHelpersUtils.get_server_type() == WebHelpersUtils.ServerType.DJANGO:
+    if WebHelpersUtils.get_server_type() == WebHelpersUtils.ServerType.DJANGO.value:

```

Comment:
thank you, if you will initialize it properly there is no need to change it 
`WebHelpersUtils.init(WebHelpersUtils.ServerType.DJANGO)`